shell/utils/validators/machine-pool.ts

```diff
@@ -0,0 +1,32 @@
+const FIELDS = {
+  NAME:     'poolName',
+  QUANTITY: 'poolQuantity'
+};
+
+const RULESETS = [
+  {
+    path:  FIELDS.QUANTITY,
+    rules: [FIELDS.QUANTITY, 'requiredInt'],
+  },
+  {
+    path:  FIELDS.NAME,
+    rules: [FIELDS.NAME],
+  }
+];
+
+export const MACHINE_POOL_VALIDATION = {
+  FIELDS,
+  RULESETS
+};
+
+export const poolName = (ctx:any) => {
+  return (name: string) => {
+    return !!name ? undefined : ctx.t('validation.required', { key: 'Value' });
```

Comment:
Can this use the pool name label instead of "Value"? Same suggestion for the quantity validator